inspirehep/modules/rt/tasks.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
```

Comment:
To me, all workflow tasks should go all together in a workflow package (or layer).
And clients of external services (like `InspireRtExt`) should go to a service layer and have a similar (or even shared) interface.

And in general these kind of refactorings are actual architectural decisions that should be discussed in a refinement. Only with the input of everyone we can take the best decision and share the knowledge.